### DISCUSSION TITLE
Why .next folder is so big? 251MB total

### DISCUSSION THREAD

**Discussion by oran1248** [vercel/next.js#17218] [Ideas]
Hi,

I'm wondering if it's ok that my .next folder's size is 251MB.
I'm building with `target: serverless`, and after running `npm run build` successfully, I looked into `.next` folder and notice that every file in `/pages` folder is ~4.5MB, also in `/api` folder. There are files that are even ~10MB.
Is it ok? Am I doing something wrong maybe?
Thanks

**Comment by Sangdt** [comment]
I think the .next folder is for caching in development and stuff. If you wanted to see the file output of the production build run "next export" after build.

**Reply by oran1248** [reply]
@Sangdt Are you sure? because when I'm running `next build` it's creating the `.next` folder

**Comment by vcexpert** [comment]
Hi.
My project's .next folder's size is too large, too.
npm run dev: 233MB
npm run build: 166MB
npm run dev and then after hours its size is increased to 9GB~13GB.
I think it is for caching in dev version but it is not normal and not good. 

How can I fix this issue?

**Comment by justinholistgr** [comment]
I encountered similar issue where the .next folder size is too large that fill up all my disk space on the server, making me unable to deploy new changes to the server via CI/CD. At the end I have to remove the folder and build manually. 

Is there any way that it could remove very old caches when running next build?

**Comment by westinde** [comment]
my .next folder is over 10gb that's wayy to big.

**Reply by nullCode666** [reply]
🐮

**Reply by Kasopej** [reply]
this really cracked me up :-D

**Comment by pooya-dehghan** [comment]
same problem here !!! its over 1.5 G after npm run build...
whats going on ?!

**Reply by GabenGar** [reply]
Only 316 MB for me with 304 MB being the cache folder.

**Reply by nullCode666** [reply]
牛

**Comment by usamaehsan** [comment]
If anyone else is facing this problem, delete your .next folder before running npm run build.
My .next folder size was 1.5gb and after deleting it , I again entered npm run build . Size reduced to just 96 mb

**Reply by miratcan** [reply]
Thank you bro!

**Reply by aliamiri1380** [reply]
still too big ...
what is it doing anyway ?

**Comment by pbalajiips** [comment]
i too have the same issue? is it ok to remove the dir .next/cache after build ?

**Reply by fabb** [reply]
i wonder the same thing: https://github.com/vercel/next.js/discussions/43666

maybe it's needed at runtime for ISR or image optimization, but i don't know for sure and would like to hear from next.js maintainers.

@timneutkens

**Comment by leerob** [comment]
Next.js can automatically create a standalone folder which copies only the necessary files for a production deployment including select files in `node_modules`: https://nextjs.org/docs/advanced-features/output-file-tracing#automatically-copying-traced-files

**Reply by bschwartz757** [reply]
@leerob Looking at the old version of our app (Next.js 7), we aren't seeing the `.next/cache` directory, which in the new version of our app (Next.js 12) is generated by `next build` and is taking up about 230MB. This is pushing the size of our build archive over the size allowed by AWS Lambda. Is there an easy way to NOT generate this directory, or to exclude it from the build artifact that needs to be uploaded to AWS Lambda? Would doing so have any negative consequences if we're not trying to use static generation/automatic static optimization?

**Reply by piyush-98** [reply]
hi @bschwartz757 i also have a similar issue after upgrading to webpack 5,due to the increase in size of cache folder inside .next. did you find any solution or cause for this and why is it happening specifically after upgrading to webpack 5

**Reply by bschwartz757** [reply]
@piyush-98 I didn't know this was a Webpack issue, my assumption was that this is a Next.js issue. In our case it seems to be the `.next/cache` folder that's responsible for most of the size. My understanding from reading the documentation is that .next/cache is generated by Next itself and is used for build caching and possibly to increase runtime performance as well. @leerob might be able to clarify whether the increased size is related to Webpack 5 or not.

**Reply by piyush-98** [reply]
Actually with webpack4 i used get a build size of 85mb and with webpack 5 it's 250+mb. I didn't change anything else in the project. By the way what happens if I delete the cache folder will it have any bad impact on the project have you tried doing this?

**Reply by bschwartz757** [reply]
@piyush-98 Interesting.... I had not tried downgrading to webpack 4, I was under the impression that Next.js 12 uses Webpack 5 by default so just went with that. I would imagine there are performance implications for downgrading the webpack version, and possibly other compatibility issues as well. In any case we have tried manually deleting the cache folder (or in our case, excluding it from our build artifact) which drastically reduces the overall artifact size but the deployed application doesn't work without it.

**Reply by leerob** [reply]
You're using output: standalone and the application is still 230mb without node_modules, is that correct?

**Reply by timneutkens** [reply]
Note that if you use output standalone the only thing you need to deploy is `.next/standalone`, nothing else. 

https://nextjs.org/docs/advanced-features/output-file-tracing#automatically-copying-traced-files

> This will create a folder at .next/standalone which can then be deployed on it's own without installing node_modules.

I guess the docs can be expanded to explain that it's about only deploying that directory, not other parts of `.next`.

**Reply by timneutkens** [reply]
> Actually with webpack4 i used get a build size of 85mb and with webpack 5 it's 250+mb.

This is entirely expected, with webpack 4 there was no caching at all, so every build would be a cold build. With webpack 5 nearly every part of the webpack build is cached to ensure subsequent builds are significantly faster.

**Reply by bschwartz757** [reply]
@leerob @timneutkens Thanks for weighing in! Actually, with `output: standalone` in our next.config it's not generating the .next/standalone directory at all - it doesn't seem to be doing anything. I get that we would want to just deploy that directory and in the meantime we're basically trying to do that manually, so it would be great if that worked but right now it's just not working at all for us.

Here's our next.config.js for reference:
```
var getGraphqlApi = require('./serverless/get-graphql-api.js')

const nextConfig = {
  output: 'standalone',
  images: {
    domains: ['companyname.scene7.com']
  },
  webpack: function (cfg) {
    const originalEntry = cfg.entry

    cfg.entry = async () => {
      const entries = await originalEntry()

      if (
        entries['main.js'] &&
        !entries['main.js'].includes('./polyfills.js')
      ) {
        entries['main.js'].unshift('./polyfills.js')
      }

      return entries
    }

    /**
     * Keeping IE11 in mind we have to forcefully transpile few external moudles from es6 to es5
     * TODO:- Remove this configuration when we decide to stop IE11 support
     */
    cfg.module.rules.push({
      test: /\.m?js$/,
      include: /node_modules\/(proprietary-form-lib|graphql|react-redux)/,
      use: {
        // Use `.swcrc` to configure swc
        loader: 'swc-loader'
      }
    })

    return cfg
  },
  publicRuntimeConfig: {...},
  env: {...}
}

module.exports = nextConfig
```

Thanks!

**Reply by bschwartz757** [reply]
@leerob @timneutkens I should clarify: we added `output: 'standalone'` and then ran `next build` as usual - is this the correct command?

**Reply by leerob** [reply]
What version of Next.js are you using? This was a recent addition.

**Reply by bschwartz757** [reply]
@leerob We're using Next@12.1.6. Which version was this added in, and is it officially supported or experimental?

**Reply by leerob** [reply]
12.2 is when the [option was stabilized](https://nextjs.org/blog/next-12-2#:~:text=In%20our%20continued,docker%20example.).

> In our continued effort to improve self-hosting Next.js, we are stabilizing our experimental outputStandalone: true config to output: 'standalone'. This config reduces deployment sizes drastically by only including necessary files/assets, including removing the need for installing all of node_modules in the built deployment package. This config can be seen in action in our [with-docker example](https://github.com/vercel/next.js/blob/canary/examples/with-docker/README.md).

**Reply by bschwartz757** [reply]
@leerob Thanks so much! We were able to get this working with `experimental.outputStandalone` in version 12.1.6, we'll upgrade to 12.2.2. 

The .next/standalone directory is much smaller, as mentioned. Unfortunately it doesn't seem to contain node modules for our custom server - but that's to be expected. I think we'll need a manual step for that until we can refactor out the custom server. The good news in our case is that we are now able to get under the Lambda size limit. 

Thanks again for the help!

**Reply by lonffreytu** [reply]
> Next.js can automatically create a standalone folder which copies only the necessary files for a production deployment including select files in `node_modules`: https://nextjs.org/docs/advanced-features/output-file-tracing#automatically-copying-traced-files

Thx, bro

**Reply by sakird** [reply]
How can we use .env variables with output: 'standalone'?

**Comment by Darginec05** [comment]
After running `next dev` my folder `.next/cache` -> 7gb
As a temporary fix I disabled cache for `next build` in the **next.config.js** file `if (!dev) config.cache = false`.
For the dev version I just wrote script to remove large .pack files in `.next/cache/webpack` folder.
And this script is located in GSP and running only in dev version.

Weird, but as a temporary fix :/
Waiting for the good solution

**Comment by tommy31** [comment]
Hey everyone, 

any update on this issue? We are facing the same problem... is this a normal behaviour? 

We ended up building our app in standalone mode. But there is a lack of documentation on this `./next/cache/webpack/` directory, we still didn't understand it's purpose.

Thanks for readings.
Tom

**Reply by shriekdj** [reply]
I have found the issue in the documentation of nextjs at here [https://nextjs.org/docs/app/building-your-application/optimizing/memory-usage#disable-webpack-cache](https://nextjs.org/docs/app/building-your-application/optimizing/memory-usage#disable-webpack-cache)

Why issue occures is on the same page [https://nextjs.org/docs/app/building-your-application/optimizing/memory-usage#webpack-build-worker](https://nextjs.org/docs/app/building-your-application/optimizing/memory-usage#webpack-build-worker)

For direct solution you can just use this code.

```js
/** @type {import('next').NextConfig} */
const nextConfig = {
  webpack: (
    config,
    { buildId, dev, isServer, defaultLoaders, nextRuntime, webpack }
  ) => {
    if (cfg.cache && !dev) {
      cfg.cache = Object.freeze({
        type: 'memory',
      })
      cfg.cache.maxMemoryGenerations = 0
    }
    // Important: return the modified config
    return config
  },
}
 
export default nextConfig
```

If it still giving you issues you can dm me on twitter [https://x.com/shriekdj](https://x.com/shriekdj)

**Comment by OssiPesonen** [comment]
Thanks a lot for all the comments in this discussion! I had an issue deploying my Next.js hybrid app to Azure Static Web Apps, because the cache folder in the build folder ended up taking 90% of the size, exceeding the > 100MB limit. By adding a removal for that folder, I managed to fix the deployment issue.

Hopefully, there will be a more permanent solution to this in the context of automated deployments.

**Reply by timneutkens** [reply]
> Hopefully, there will be a more permanent solution to this in the context of automated deployments.

Have a look at https://github.com/vercel/next.js/discussions/17218#discussioncomment-3089184

**Reply by adarsh-drishya** [reply]
i am facing this issue the standalone folder is way too large, more than 300 mb which exceeds the limit of 200 mb to deploy to aws amplify instance. @timneutkens

**Comment by mcmxcdev** [comment]
There seems to be no solution for this after more than two years of discussion.

Can we at least have the cache size limited to a maximum and make it configurable? A coworker ended up with a size of 300GB+ just for the cache folder... My SSD is only 128GB...

**Reply by Darginec05** [reply]
Could you show `package.json` and how many dynamic imports do you have in project?

**Comment by Darginec05** [comment]
SOLVED.

So, as I wrote earlier, my cache folder took up about >= 7 GB in development mode.
This usually happens if you have multiple dynamic imports (my application has 123 dynamic imports of large components).
I noticed that the fragments used dependent packages from node_modules.
So problem can be found [here](https://github.com/vercel/next.js/blob/canary/packages/next/build/webpack-config.ts#L1438).

Option for webpack optimization `splitChunks` setted to false in dev mode.
As a solution I added these changes to my next.config.js:

`if (dev) {
      config.optimization.splitChunks = {
        cacheGroups: {
          framework: {
            chunks: 'all',
            test: /[\\/]node_modules[\\/]/,
            name: 'framework',
            enforce: true,
          },
        },
      };
    }`

**Reply by Darginec05** [reply]
So, node_modules splitted into separate chunk and cache size after these changes is ±900MB in dev.

**Reply by SuperOleg39** [reply]
Good catch, sad that I found that comment so late.

Investigate and made a similar optimization in our projects with webpack few month ago, and trying to share this experience with next team - https://github.com/vercel/next.js/issues/49929#issuecomment-1647549143

Interesting, that splitChunks was added to dev server build recently - https://github.com/vercel/next.js/pull/54988

@timneutkens hi! Maybe you know some significant disadvantages of adding splitChunks to client Dev build?

**Comment by Drillman** [comment]
I came across the same issue of a big .next/cache folder, and it was making my Docker image way too big. After searching a bit I found [this answer](https://github.com/vercel/next.js/discussions/31395#discussioncomment-1643499), the cache folder is only used for building purposes and is not needed for production nor running the app in general it seems.

**Reply by shriekdj** [reply]
For Your Specific Docker related issue this section of nextjs docs can help you [https://nextjs.org/docs/app/building-your-application/deploying#configuring-caching](https://nextjs.org/docs/app/building-your-application/deploying#configuring-caching).

It helps stopping or customizing cache behaviour.

`cacheMaxMemorySize: 0` will disable the in memory ( RAM ) caching size which was 50 mb

`cacheHandler: require.resolve('./cache-handler.js')` wil need to configure your cache. there is `cache-handler` docs for it.

**Reply by shriekdj** [reply]
you can use `cache-handler.js` for your other cache handler programs like redis or something else

**Comment by Stroller15** [comment]
can anyone tell me how you are knowing your folder sizes??
thanks

**Reply by fabb** [reply]
You could use the disk usage tool:
https://en.m.wikipedia.org/wiki/Du_(Unix)

**Reply by Stroller15** [reply]
Thanks man!

**Comment by fungilation** [comment]
Upgrading from Next 11 to 14 here. What observation I see is build cache >1gb and thus Vercel can no longer cache it
```
Build cache size exceeds limit (1.00 GB). Can not store build cache.
```

On Next 11, my builds used to have 400mb cache. Now:
```
du -h .next/cache
  0B	.next/cache/swc/plugins/v7_macos_x86_64_0.104.41
  0B	.next/cache/swc/plugins
  0B	.next/cache/swc
465M	.next/cache/webpack/client-production
306M	.next/cache/webpack/edge-server-production
305M	.next/cache/webpack/server-production
1.1G	.next/cache/webpack
1.1G	.next/cache
```

I have 1 function which is "server-rendered on demand using the Edge Runtime". Many others are on Node runtime. So why the near duplication between the edge-server-production and server-production cache folders?

This also leads to build being slow, if webpack is building twice for every function, 1 for Edge and 1 for Node, regardless of how the api functions are actually deployed. So it's a double whammy, wasting build time and cache size (exceeding 1gb in my case)

Can someone from Vercel / Next shed some light please

**Reply by kcblaq** [reply]
I am currently stuck on same issue, size now is 1.6g and I can no longer work with git.

I will appreciate help ASAP.

**Reply by fungilation** [reply]
@leerob ?

**Reply by Jaimayal** [reply]
Just found mine goes over 6gb:
4.6G    .next/cache/webpack/server-production
11M    .next/cache/webpack/edge-server-production
1.6G    .next/cache/webpack/client-production
6.2G    .next/cache/webpack

I barely have any images, so I believe my issue is similar to yours. Have you tried disabling webpack cache?
https://nextjs.org/docs/app/building-your-application/optimizing/memory-usage#disable-webpack-cache

Edit: I'm on Next 14.0.4

**Comment by Diluka** [comment]
I have a 5+GB `.next/cache/images`

**Reply by Diluka** [reply]
It's grow from 0 to 5GB during standalone app running.

**Reply by shriekdj** [reply]
it's because of your public folder have too many images.

When we run `next build` it analyzes all the images in your `public` folder first too cache it then it builds project completely.

Here I think you must also facing the server startup taking too much time from normal for production. I had the same issue so I am now showing images from our local cdn domain.

**Reply by shriekdj** [reply]
You can disable or stop this behavior with below code as per [https://nextjs.org/docs/pages/api-reference/components/image#minimumcachettl](https://nextjs.org/docs/pages/api-reference/components/image#minimumcachettl)

with updating `next.config.js` where

```js
images: {
  minimumCacheTTL: 60
  // it can 0 also for disabling
}
```

**Reply by shriekdj** [reply]
also it creates 2 formats for optimising images sometimes `'image/avif' | 'image/webp'`

You can set it to use only single one which like this

```js
images: {
  formats: [ 'image/webp' ]
}
```

**Comment by shriekdj** [comment]
this is because of the default cache created by the nextjs

For Resolving FetchCache Issue

You can add this at root `layout.tsx`

```tsx
export const fetchCache = 'no-store'
// same level as metaData and RootLayout


// or Adding revalidate of 10 second or 5 minutes ( 300 seconds )

export const revalidate = 300
```

For Solving issue of Image Cache add this in `next.config.js` where `minimumCacheTTL` is number of seconds to cache the images

```js
images: {
  minimumCacheTTL: 86400
}
```

**Comment by kamran1381** [comment]
the .next file size increases every time npm  run dev is performed. saw something about deleting the cache folder before deploying it, but the cache folder is just going to come back after you run your server again. it seems like there is no permanent solution of containing the cahce folder size in next 14 yet :(

**Comment by syedusama5556** [comment]
you can just delete the .next/cache folder and then deploy it it works fine and the size is reduced from 1.2GB to 9.5MB and we are running it using PM2 on the live server without any issue.

**Reply by fungilation** [reply]
nope, doesn't work. I can delete .next/ all I want, 1.1G comes back every time on build. Local or on Vercel

https://github.com/vercel/next.js/discussions/17218#discussioncomment-8648431

**Reply by syedusama5556** [reply]
yes you have to delete it after every successful build and you have to use npm run start instead of dev on live server, also note first you have to understand the cache will not be gone forever on live server also as nextjs caches the components or routes that are frequently used so you will see some new caching it wont me in GBs but much smaller. if you dont want any cache you can create a cronjob in Linux server and run it preodically to check if size of cache folder is exceeding a certain amount then remove it

**Comment by vipinyadav2k** [comment]
So we dont need node_modules files right ? As that is also somewhat 500mb

**Reply by GabenGar** [reply]
What do you mean by "don't need"? That's literally the folder where all dependencies of your code, including nextjs, are stored.

**Reply by vipinyadav2k** [reply]
nvm i was able to deploy my project without it and it works fine.

**Reply by shriekdj** [reply]
@GabenGar  , Maybe he was talking about the static export

**Reply by adnan-smlatic** [reply]
Peak indian software development

**Comment by SouthLink** [comment]
I'm using a custom server, but unfortunately it's in standalone mode and server.js can't be tracked. 

https://nextjs.org/docs/pages/building-your-application/configuring/custom-server
*Standalone output mode, does not trace custom server files and this mode outputs a separate minimal server.js file instead.*

So I removed.next/cache and reduced the build cache by 1.5GB，But I can't remove the node_module， this my app dockerfile:

```
FROM node:16-slim as next-base

########                install + builder                 ########
FROM next-base AS installBuilder

WORKDIR /www

# Copy package.json
COPY ./package.json /www/package.json
COPY ./.npmrc /www/.npmrc
COPY ./pnpm-lock.yaml /www/pnpm-lock.yaml 

RUN  npm install webpack -g && npm install -g pnpm@8.7.5 && pnpm install

COPY . /www

# delete cache，It does not affect application operation
RUN pnpm run build:test  && rm -rf /www/.next/cache

########              runner                ########
FROM next-base AS runner

RUN groupadd www -g8080
RUN useradd www -u8080 -g8080

WORKDIR /www

RUN apt-get update && apt-get install nginx git -y

# pm2 
RUN npm install pm2@latest -g
RUN pm2 update
RUN pm2 install pm2-intercom

COPY --from=installBuilder /www ./

COPY ./_build/nginx_develop.conf /etc/nginx/nginx.conf

CMD pm2 start ecosystem.config.js --env test && nginx -g 'daemon off;'
```

**Comment by lakshya-webDev** [comment]
I face the same issue it have reduced to 800mb but still I want to know if there is any quicky solution for this size issue 

```

/ Import necessary modules and configuration
const withPlugins = require("next-compose-plugins");
const withBundleAnalyzer = require("@next/bundle-analyzer")({
  enabled: process.env.ANALYZE === "true",
});
// Define the Next.js configuration
const nextConfig = {
  reactStrictMode: false,
  future: {
    webpack5: true,
  },
  compress: true,
  eslint: {
    ignoreDuringBuilds: true,
  },
  images: {
    minimumCacheTTL: 86400,
    formats: ["image/webp"],
    remotePatterns: [
      {
        protocol: "https",
        hostname: `${process.env.NEXT_PUBLIC_STRAPI_URL}`,
        port: "",
      },
    ],
  },
  webpack: (config, { buildId, dev, isServer, defaultLoaders, webpack }) => {
    // Modify webpack config if necessary
    return config;
  },
};

// Export the configuration using next-compose-plugins
module.exports = withPlugins([withBundleAnalyzer], nextConfig);

```

**Reply by shriekdj** [reply]
I have found the issue in the documentation of nextjs at here [https://nextjs.org/docs/app/building-your-application/optimizing/memory-usage#disable-webpack-cache](https://nextjs.org/docs/app/building-your-application/optimizing/memory-usage#disable-webpack-cache)

Why issue occures is on the same page [https://nextjs.org/docs/app/building-your-application/optimizing/memory-usage#webpack-build-worker](https://nextjs.org/docs/app/building-your-application/optimizing/memory-usage#webpack-build-worker)

For direct solution you can just use this code.

```js
/** @type {import('next').NextConfig} */
const nextConfig = {
  webpack: (
    config,
    { buildId, dev, isServer, defaultLoaders, nextRuntime, webpack }
  ) => {
    if (cfg.cache && !dev) {
      cfg.cache = Object.freeze({
        type: 'memory',
      })
      cfg.cache.maxMemoryGenerations = 0
    }
    // Important: return the modified config
    return config
  },
}
 
export default nextConfig
```

If it still giving you issues you can dm me on twitter [https://x.com/shriekdj](https://x.com/shriekdj)

**Comment by holedev** [comment]
I deploy to Vercel and get similar error:

![image](https://github.com/user-attachments/assets/95cea7a1-19f7-42e5-9440-607982122ba1)

Reading at [https://nextjs.org/docs/app/api-reference/config/next-config-js/output#caveats](https://nextjs.org/docs/app/api-reference/config/next-config-js/output#caveats), about `outputFileTracingExcludes`. I have add the code below in `next.config.ts`:

`
const nextConfig: NextConfig = {
  ...,
  outputFileTracingExcludes: {
    "/api/docs": ["./.next/cache/**/*"]
  },
  ....
};
`

Then the project working right, my nextjs version is `15.1.6`, you can check my config at [my project](https://github.com/holedev/starter-kit). Hope this help everyone.

**Comment by Ahsan2001** [comment]
Step 1:  Update your next.config.ts to this file 

import type { NextConfig } from 'next';
const nextConfig: NextConfig = {
  productionBrowserSourceMaps: false,
  output: 'standalone',
};
export default nextConfig;



Step 2: install this package 
 npm install --save-dev rimraf


Step 3: replace you build script in package json
 "build": "next build && rimraf .next/cache",

**Comment by bnmghjtyu0** [comment]
Opt1
1. npm install -g vercel
2. vercel build --prod
3. rm -rf .next/cache
4. vercel deploy --prebuilt --prod

Opt2
1. npm install -g vercel
2. npm install rimraf
3. package.json
```
{
  "vercel:deploy": "vercel build --prod && npx rimraf .next/cache && vercel deploy --prebuilt --prod"
}
```
4. npm run vercel:deploy

**Comment by MikeCarbone** [comment]
Running into this problem too. Cache in dev grows to > 30GB over time. Changing my `dev` command to this: 
```
"dev": "NODE_OPTIONS='--max-old-space-size=4096' next dev --turbo",
```
automatically restarts the server once memory hits 4GB 🤷‍♂️

**Comment by MikeOuroumis** [comment]
Temporarily setting the build command in Vercel to `rm -rf .next/cache && next build` worked for me.
![Screenshot 2025-06-01 at 13 25 36](https://github.com/user-attachments/assets/03a74343-a8b6-417c-9669-2de0db062f6d)

**Comment by raghavendra-k-j** [comment]
The reason is they have monopoly over React and SSR Framework.

I think there are doing som technical tackticks with Technical peoples.

I started with Next JS and moved to Remix and Because of these peoples. I tried on POC of Custom SSR rendering, It worked, so I Completed moved  to React with Vite Build tool.

You can also use Remix - but it has many issues at begining.

Most of the times you do no need SSR. Go with React and Vite

**Comment by pgniadek** [comment]
I have a next js application running in production. I use app router and ISR to cache pages. After, for example, a month after deploying the app, the .next folder can take up tens of GB. The .next/server/app folder is mainly responsible for this size. 

Does anyone know what I could do to prevent this folder from reaching such a large size? I don't need next js to hold the page cache indefinitely. Could I for example delete files older than 3 days from it while the application is still running production without risking breaking the application?
